Jenkinsfile

```diff
@@ -1,8 +1,8 @@
 #!groovy
 
-def python_versions = ['2.7','3.3','3.4','3.5','3.6','pypy','latest']
+def python_versions = ['python:2.7','python:3.3','python:3.4','python:3.5','python:3.6','pypy:2', 'pypy:3','python:latest']
```

Comment:
I think we can omit `python:latest`, it's essentially the same as `python:3.6`. Unfortunately, there isn't a docker image for nightly Python master builds, but it's not suuper important to have those.

Reply:
will do